inst/include/RcppArmadilloAs.h

```diff
@@ -403,14 +403,61 @@ namespace traits {
                     res = symmatl(res);
                 }
             }
+            else if (type == "pMatrix") {
+                IntegerVector i(ncol);
+                IntegerVector p(ncol+1);
```

Comment:
For now, I think the pMatrix might not be converted to dgCMatrix since different pMatrix can lead to the same dgCMatrix, but if you have any ideas, please let me know. I'll add warning messages in the conversion of pMatrix.